config/_default/languages.toml

```diff
@@ -12,15 +12,14 @@
    languageCode = "zh-Hans"
    contentDir = "content/zh"
 
-   [zh.params]
-    description = ""
+    
   [[zh.menu.main]]
   name = "主页"
   url = "#home_slider"
   weight = 1
 
   [[zh.menu.main]]
-  name = "简介"
+  name = "关于"
```

Comment:
Your local files seems to be old version, please synchronize the master changes of this file to your local.